apps/prairielearn/src/ee/lib/validateHTML.ts

```diff
@@ -0,0 +1,616 @@
+import * as parse5 from 'parse5';
+
+const HTML_GLOBAL_TAGS = [
```

Comment:
These are attributes, not tags, right?

Reply:
Changed in 0e87e7e4333c2d4d8dfbc2116aad0c0acd81707b.